### ISSUE TITLE
Action : Allocation fails with exception error

### ISSUE THREAD

**Issue by yuwtennis** [elastic/curator#871]
Curator version : elasticsearch-curator (4.2.5)
Elasticsearch version :elasticsearch-5.1.1

I was testing the curator setting using the configuration from this [blog](https://www.elastic.co/blog/hot-warm-architecture-in-elasticsearch-5-x). However, allocation action dies with exception error.

```
2017-01-25 17:36:25,427 ERROR curator.cli cli:193 Failed to complete action: allocation. <class 'curator.exceptions.FailedExecution'>: Exception encountered. Rerun with loglevel DEBUG and/or check Elasticsearch logs for more information. Exception: TransportError(400, u'illegal_argument_exception', u'wait_for_relocating_shards has been removed, use wait_for_no_relocating_shards [true/false] instead')
```

As the error shows , [code](https://github.com/elastic/curator/blob/8fd6863d8b401c39524fb3dcc8c1fdb49ffa85b1/curator/actions.py) uses wait_for_relocating_shards .
According to the [release note](https://www.elastic.co/guide/en/elasticsearch/reference/5.x/breaking_50_rest_api_changes.html#_remove_properties_support) option name had changed in 5.1.

**Comment by willyaranda** [comment]
Having the same problem here with a clean instalation of curator + elastic

**Comment by untergeek** [comment]
Chances are _very_ good that this will not be repaired in Curator 4.x, but will be in 5.x.  It will be documented in the 4.x tree (soonish) that this feature does not work when using Curator 4.x with 5.x

This is what makes cross-version support hard 🙁

**Comment by untergeek** [comment]
Fixed by #896
